shell/app/modules/org/pages/setting/notice-channel.tsx

```diff
@@ -263,19 +316,29 @@ const NotifyChannel = () => {
                 'please input the SMS Template Code you have applied for on the service provider platform',
               )}`}
             />
-            <div className="text-desc mt-4">
-              {i18n.t('Submit the following information to the service provider to apply for an SMS template')}:
-            </div>
-            <div className="text-desc mt-2">
-              <Copy
-                copyText={`${i18n.t('You have a notification message from the Erda platform')}: $\{content}, ${i18n.t(
-                  'please deal with it promptly',
-                )}`}
-              >
-                {`${i18n.t('You have a notification message from the Erda platform')}: $\{content}, ${i18n.t(
-                  'please deal with it promptly',
-                )}`}
-              </Copy>
+            <div className="bg-grey px-2 py-3 rounded-sm mt-2">
+              <div className="text-sub">
+                {i18n.t('Submit the following information to the service provider to apply for an SMS template')}:
+              </div>
+              <div className="mt-2">
+                <span className="bg-white text-normal p-1 pr-2 font-semibold">{`${i18n.t(
+                  'You have a notification message from the Erda platform',
+                )}: $\{content}, ${i18n.t('please deal with it promptly')}`}</span>
+                <span
+                  className="text-primary cursor-pointer underline ml-2 jump-to-aliyun"
+                  data-clipboard-text={`${i18n.t(
+                    'You have a notification message from the Erda platform',
+                  )}: $\{content}, ${i18n.t('please deal with it promptly')}`}
+                  onClick={() =>
+                    window.open(
+                      'https://account.aliyun.com/login/login.htm?oauth_callback=https%3A%2F%2Fdysms.console.aliyun.com%2Fdysms.htm%23%2Fdomestic%2Ftext%2Ftemplate%2Fadd',
+                    )
+                  }
+                >
+                  {i18n.t('copy and jump to the application page')}
+                </span>
+                <Copy selector=".jump-to-aliyun" />
```

Comment:
click tigger copy and jump？

Reply:
![image](https://user-images.githubusercontent.com/30014895/141096686-03ba6ca3-d11e-4603-9d79-5859d98f7e3a.png)


Designer and back-end demand the interactive, copy the template and jump to aliyun and paste it in some place